boot/zephyr/main.c

```diff
@@ -193,8 +213,8 @@ void main(void)
     rc = gpio_pin_read(detect_port, CONFIG_BOOT_SERIAL_DETECT_PIN,
                        &detect_value);
     __ASSERT(rc == 0, "Error of the reading the detect pin.\n");
-
-    if (detect_value == CONFIG_BOOT_SERIAL_DETECT_PIN_VAL) {
+    if (detect_value == CONFIG_BOOT_SERIAL_DETECT_PIN_VAL && \
```

Comment:
Sorry, I know that's copy/pasted from my code, but I had python on the brain... backslash at end of line is not needed :)